### PR TITLE
[FIX] web_responsive: Traceback on grouped searchpanel

### LINKED ISSUE #2016: web_responsive: search panel with groupby fails at low screensize
On v14 if the search panel has a field with groupby on, and the screensize is low, the action load fails with the following error message: 

```
TypeError: filter.groups.values.map is not a function or its return value is not iterable
    at PatchableClass.getActiveSummary (https://odoo14.hatter.hu/web/content/350-b1a1a09/web.assets_backend.js:6081:410)
    at QWeb.eval (eval at _compile (https://odoo14.hatter.hu/web/content/229-51c67ad/web.assets_common.js:969:178), <anonymous>:29:50)
    at QWeb.render (https://odoo14.hatter.hu/web/content/229-51c67ad/web.assets_common.js:958:20)
    at __owl__.renderFn (https://odoo14.hatter.hu/web/content/350-b1a1a09/web.assets_backend.js:1332:147)
    at Object.__owl__.renderFn (https://odoo14.hatter.hu/web/content/229-51c67ad/web.assets_common.js:1308:113)
    at PatchableClass.__render (https://odoo14.hatter.hu/web/content/229-51c67ad/web.assets_common.js:1288:33)
    at https://odoo14.hatter.hu/web/content/229-51c67ad/web.assets_common.js:1252:6
```

### PR DESCRIPTION
**Pull request by SplashS**
Fixes #2016

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: web_responsive/static/src/js/web_responsive.js [hunk 1]
```diff
@@ -599,7 +599,7 @@ odoo.define("web_responsive", function (require) {
                         let values = [];
                         if (filter.groups) {
                             values = [
-                                ...filter.groups.values().map((g) => g.values),
+                                ...[...filter.groups.values()].map((g) => g.values),
```
Comment (Tardo) [inline web_responsive/static/src/js/web_responsive.js]: We could make it a little more readable by using...
`values = Array.from(filter.groups.values(), item => item.values)`

How do you see it? Anyway, thanks for the patch.

**Comment by ITerraGroup** [comment]
@Tardo thanks, it's clearer.

File: web_responsive/static/src/js/web_responsive.js (hunk 1 above)
Comment (SplashS) [reply web_responsive/static/src/js/web_responsive.js]: Thanks, it's clearer.
Tested this variant on runbot. It works.

**Pull request merged**